src/test/resources/net/sf/jabref/importer/fileformat/MedlineImporterTestArticleID.xml

```diff
@@ -30,7 +27,7 @@
             </Journal>
             <ArticleTitle>Diagnostic performance of a multiplex PCR assay for meningitis in an HIV-infected population in Uganda.</ArticleTitle>
             <Pagination>
-                <MedlinePgn/>
+            <MedlinePgn>1234</MedlinePgn>
```

Comment:
I'm not sure from where the xml file comes. This file was made in an earlier pull request by @mairdl .
But like it was before, the document wasn't parsed, because it wasn't even well formed (closing tag, but no opening tag). That's why I changed it.